packages/core/integration-tests/test/transpilation.js

```diff
@@ -120,7 +120,7 @@ describe('transpilation', function () {
     await bundle(path.join(__dirname, '/integration/jsx-react-alias/index.js'));
 
     let file = await outputFS.readFile(path.join(distDir, 'index.js'), 'utf8');
-    assert(file.includes('React.createElement("div"'));
```

Comment:
This test case seems wrong, This test case seems wrong. In this test case, `React.createElement` should be transpiled into `h`.

Reply:
Are we sure this is right? It should map to `preact/compat` in this case, which should have `createElement` (and may be different than pure Preact for compatibility reasons).

Reply:
Oh I see ! I’m sorry I misunderstood :bow:
As you say, `preact/compat` exports `createElement` and does not `h` and `React` is aliased to `preact/compat`, so `React.createElement` should be transpiled into `React.createElement`, so this test case is not wrong.

Reply:
09b9844 👍